### ISSUE TITLE
[Enhancement] Check file system permissions

### ISSUE THREAD

**Issue by joendres** [emilsvennesson/script.module.inputstreamhelper#45]
TL;DR
libwidevinecdm.so might end up on a file system that's mounted noexec and hence not being loadable. A check or even automatic fix would be nice.

Long version

My libreelec on raspi2 didn't have enough disk space to download the crome recovery image. So I added as USB stick an symlinked the script.module.inputstream directory to it. The automatic installation of libwidevine via Netflix addon then worked jaust fine, but the library was unusable, because the USB stick was mounted noexec. The symptom was that Netflix Videos would just not start - without any error message. 

The relevant message in kodi.log is
 ERROR: AddOnLog: InputStream Adaptive: Unable to load widevine shared library (/storage/.kodi/cdm/libwidevinecdm.so)

**Comment by joendres** [comment]
#20 would solve the actual problem, but getting an error message would still be apprecieated.

**Comment by joendres** [comment]
8872212a44eb50e730c6cda4f03e3ca33016b6a0 solved my initial problem, but since it's not in a release yet, my libreelec didn't get it as an update ;-)

**Comment by dagwieers** [comment]
@joendres This has now been release as part of v0.3.5
I am going to close this issue, if you still have problems, feel free to reopen or provide feedback.
